.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Don't disable the linter! Please remove this file.

Reply:
Oh, looks like this is enabling the linter. In that case, lets put this in the `.gitignore` file since these are editor specific settings.